library/debug.c

```diff
@@ -69,6 +70,10 @@ void mbedtls_debug_print_msg(const mbedtls_ssl_context *ssl, int level,
     char str[DEBUG_BUF_SIZE];
     int ret = MBEDTLS_ERR_ERROR_CORRUPTION_DETECTED;
 
+#if defined(static_assert)
```

Comment:
That's never going to trigger without `#include <assert.h>`.

Reply:
Bah, good spot - thanks